examples/nlp/README.md

```diff
@@ -0,0 +1,72 @@
+﻿# Natural Language Processing with ZenML
+
+This example introduces a standard Natural Language Processing (NLP) workflow in ZenML.
+
+The NLP ecosystem of choice for this example is [HuggingFace](https://github.com/huggingface). They provide many
+convenience utilities such as tokenizers, models and datasets for use in pipelines. We want to showcase here how easy it
+is to integrate these tools into a production ML setting with ZenML.
+
+## Pre-requisites
+
+In order to run this example, you need to clone the zenml repo.
+
+```bash
+git clone https://github.com/maiot-io/zenml.git
+```
+
+Before continuing, either [install the zenml pip package](https://docs.zenml.io/getting-started/installation.html) or
+install it [from the cloned repo](../../zenml/README.md). In both cases, make sure to also install the gcp extension (
+e.g. with pip: `pip install zenml[gcp]`)
+
+```
+cd zenml
+zenml init
+cd examples/gcp_dataflow_processing
+```
+
+Also, you have to install some HuggingFace libraries to be able to proceed. To do that, in your ZenML virtual
+environment run
+
+```
+pip install transformers tokenizers
```

Comment:
maybe pin a version here? Just in case

Reply:
Resolved by adding the Huggingface extras_require installation in 43c6ab5